src/Microsoft.AspNetCore.Mvc.Core/Authorization/AuthorizeFilter.cs

```diff
@@ -13,12 +13,13 @@
 namespace Microsoft.AspNetCore.Mvc.Authorization
 {
     /// <summary>
-    /// An implementation of <see cref="IAsyncAuthorizationFilter"/>
+    /// An implementation of <see cref="IAsyncAuthorizationFilter"/> which applies a specific
+    /// <see cref="AuthorizationPolicy"/>.
     /// </summary>
```

Comment:
I would add somewhere here that this is the filter that's added by the framework where you use `[Authorize(...)]`